web-extensions/chrome/scripts/load-bookmarks.js

```diff
@@ -141,11 +191,144 @@ function addEvents(mangaList) {
+            bmcEngine.removeEventListener(bmcEngine.events.register.complete, handleRegisterSuccess);
+            mangaList.setMode(mangaList.MODE_BROWSE);
+            showHideSidePanel();
+            notifyResult('Register Comic', new Error('Could not alias current page to comicId ' + comicId));
+        };
+        const handleRegisterSuccess = () => {
+            bmcEngine.removeEventListener(bmcEngine.events.register.error, handleRegisterError);
+            bmcEngine.removeEventListener(bmcEngine.events.register.complete, handleRegisterSuccess);
+            mangaList.setMode(mangaList.MODE_BROWSE);
+            showHideSidePanel();
+            notifyResult('Register Comic');
+        };
+        bmcEngine.addEventListener(bmcEngine.events.register.error, handleRegisterError);
+        bmcEngine.addEventListener(bmcEngine.events.register.complete, handleRegisterSuccess);
+        // Don't need to provide the reader/name/chapter/pages, as they're
+        // already set in the engine since the instanciation.
+        bmcEngine.register(label);
+    };
+
+    bmcEngine._messaging.addWindowHandler(
+        bmcEngine.SIDEPANEL_ID,
+        evData => evData.type === 'action' && evData.action === 'notification',
+        evData => {
+            console.log(`BmcSidePanel: received message to display status notification op=${evData.operation} err=${evData.error}`);
+            notifyResult(evData.operation, evData.error);
+        });
 }
 
 var mangaList = new BmcMangaList();
 mangaList.generate();
 addEvents(mangaList);
 
+
+/*
+ * This function ensures the transition CSS class is absent, changes the
+ * background color (static, instantaneous).
+ * Then, it applies the transform class, and finally changes the color back to
+ * the original color. This creates a smooth transform from the provided color
+ * parameter to the original color.
+ */
+function triggerTransition(elem, color) {
+    // Ensure we get the final computed one (from CSS sheet), as using `.style`
+    // only accesses inline values.
+    const origColor = getComputedStyle(elem).backgroundColor;
+
+    // -> Ensure it's not present when setting the color
+    elem.classList.remove('notif-transform');
+
+    // Now set the color to transition from
+    elem.style.backgroundColor = color;
+
+    /*
+     * /!\ NOTE IMPORTANT /!\
+     * Accessing a DOM property seems to force a redraw, preventing browser
+     * optimization on style settings (by batching updates) which might
+     * actually hide the transition.
+     * /!\ NOTE IMPORTANT /!\
+     */
+    void elem.offsetHeight;
+
+    // Add the transition effect _before_ changing the color
+    elem.classList.add('notif-transform');
+
+    // And finally set the color to get back to (original color)
+    // => This triggers the actual transition effect
+    elem.style.backgroundColor = origColor;
+
+    // Remove all mentions of transitions after 2 secs
+    setTimeout(() => {
```

Comment:
And that would also allow to remove this.

Reply:
similarly, to be addressed as issue #18